### DISCUSSION TITLE
Zigbee + MQTT dopo aggiornamento HA 2022.12

### DISCUSSION THREAD

**Discussion by baldus88** [indomus/forum#2168] [ZigBee, Z-Wave, Thread, Matter]
Ciao ragazzi.
Cerco di essere più preciso possibile anche se di problemi me ne sono portati dietro un po' dopo aver aggiornato da HA Core 2022.7.5 -> 2022.12.1.

Prima di fare il grande aggiornamento mi sono scorso i vari post di indomus di ogni major release per capire cosa veniva deprecato e cosa non avrebbe più funzionato. Avevo individuato che sicuramente zwave JS era da aggiornare e poco altro. 
Ho proseguito a fare l'aggiornamento complessivo... ho approfittato pure per aggiornare zigbee2mqtt.

Sta di fatto che mi sono arrivati una sbroloata di problemi che piano piano sto cercando di risolvere, come aggiornare python alla 3.10.X, rust etc
Una cosa che mi ha impattato molto è la sintassi di MQTT che cambia (da mesi in realtà) dalla platform key alla configuration key. Questa è un mea culpa essendo che l'aggiornamento è già citato nella release 2022.6.

Mi sono messo a cambiare il tutto e sono arrivato ad avere una configurazione che i binary_sensor che usano un altro protocollo (vedesi radiofrequenza) e comunicano via mqtt sono ok. Invece quelli che c'è di mezzo zigbee non funziona proprio.

![image](https://user-images.githubusercontent.com/57867040/207056563-1c91309a-8048-42ab-974c-72b3bec22909.png)

Ho provato a navigare la dashboard di zigbee2mqtt ma non vedo nulla purtroppo:

![image](https://user-images.githubusercontent.com/57867040/207054807-4a818cd7-4dfa-4f41-a329-0ce3ad6b81ee.png)

Io la usavo come punto di accesso primario per aver un integrazione automatica tra HA e Zigbee2MQTT, quindi tutti i multisensor, luci etc non me le vede più non avendo dispositivi agganciati. 

Ho provato a reinstallare il componente ma continua a non andare. 
Per darvi un contesto io uso un dongle USB (CC2538+CC2592) quello di produzione Giovanni.
Questa è la configurazione:

```yaml
homeassistant: true
permit_join: true
# MQTT settings
mqtt:
  base_topic: zigbee2mqtt
  server: 'mqtt://localhost'
  user: mqtt_user
  password: 
  version: 5
  include_device_information: true
serial:
  port: /dev/ttyUSB0
advanced:
  log_directory: /var/log
experimental: {}
frontend:
  # porta su cui verrà esposta la webGUI
  port: 8080
ota:
  update_check_interval: 10
disable_automatic_update_check: false
```
La porta seriale chiaramente è quella giusta dopo averla verificata con il rispettivo comando:
```
pi@raspberrypi:~ $ ./ports.sh | grep USB
/dev/ttyUSB0 - 1a86_USB_Serial
```

Dai log del journal non ci sono errori rilevanti:
```
Dec 12 13:03:45 raspberrypi bash[454]: Zigbee2MQTT:warn  2022-12-12 13:03:45: Allowing new devices to join.
Dec 12 13:03:45 raspberrypi bash[454]: Zigbee2MQTT:warn  2022-12-12 13:03:45: Set `permit_join` to `false` once you joined all devices.
Dec 12 13:03:45 raspberrypi bash[454]: Zigbee2MQTT:info  2022-12-12 13:03:45: Zigbee: allowing new devices to join.
Dec 12 13:03:45 raspberrypi bash[454]: Zigbee2MQTT:info  2022-12-12 13:03:45: Connecting to MQTT server at mqtt://localhost
Dec 12 13:03:45 raspberrypi bash[454]: Zigbee2MQTT:info  2022-12-12 13:03:45: Connected to MQTT server
Dec 12 13:03:45 raspberrypi bash[454]: Zigbee2MQTT:info  2022-12-12 13:03:45: MQTT publish: topic 'zigbee2mqtt/bridge/state', payload 'online'
Dec 12 13:03:46 raspberrypi bash[454]: Zigbee2MQTT:info  2022-12-12 13:03:46: Started frontend on port 0.0.0.0:8080
Dec 12 13:03:46 raspberrypi bash[454]: Zigbee2MQTT:info  2022-12-12 13:03:46: MQTT publish: topic 'zigbee2mqtt/bridge/config', payload '{"commit":"52e545f9","coordinator":{"meta":{"maintrel":2,"majorrel":2,"minorrel":7,"product":2,"revision":20200211,"transportrev":2},"type":"zStack30x"},"log_level":"info","network":{"channel":11,"extendedPanID":"0x00124b0009d6caa1","panID":6754},"permit_join":true,"version":"1.28.4"}'
Dec 12 13:03:46 raspberrypi bash[454]: Zigbee2MQTT:info  2022-12-12 13:03:46: MQTT publish: topic 'zigbee2mqtt/bridge/state', payload 'online'
Dec 12 13:03:46 raspberrypi bash[454]: Zigbee2MQTT:info  2022-12-12 13:03:46: Zigbee2MQTT started!
```

Mi potete dare una mano intanto ad uscire da questo inghippo?

Grazie a chi potrà darmi una mano.

Saluti

**Comment by snakuzzo** [comment]
A me sembra tutto perfettamente funzionante.
Da quello che vedo nello screenshot, dopo quello che hai fatto, ti sei perso la directory `data` di zigbee2mqtt e di conseguenza , se non hai un backup, mi spiace...ma devi rifare il pairing di tutti i device.

**Reply by baldus88** [reply]
Grazie mille, era più semplice del previsto. Ora alcuni sensori riesco a metterli altri no, ma devo combattere come sempre con il pairing oramai ci sono abbonato.

Avevo in realtà comunque pure salvato la cartella data ma il file database.db non lo prende a quanto pare

**Reply by snakuzzo** [reply]
permessi sbagliati forse ?

**Reply by baldus88** [reply]
Rettifico gl'ho ripassato la cartella intera ora me li vede tutti sulla dashboard ma gli input non li prende, misà che li rimetto da capo almeno vedo mano mano cosa non va :)
